### ISSUE TITLE
Page Header/Footer

### ISSUE THREAD

**Issue by jaceee** [nRewik/SimplePDF#3]
Can this feature be implemented in the near future?
I think I could throw a weekend at it and ask for a pull request if needed.

**Comment by nRewik** [comment]
Yes. It's quite simple to implement that feature. And PR is welcome :+1: .

**Comment by caloon** [comment]
Any updates on this?

**Comment by davidseek** [comment]
+1

**Comment by daviddelmonte** [comment]
+1

**Comment by cerupcat** [comment]
@nRewik Any recommendations on how to go about adding this?
